media-gfx/img2pdf/img2pdf-0.4.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="Losslessly convert raster images to PDF"
+HOMEPAGE="https://gitlab.mister-muffin.de/josch/img2pdf"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# missing dev-python/pdfrw dependency
+# require old version of imagemagick
+# some test fails without patch
+# see https://gitlab.mister-muffin.de/josch/img2pdf/issues/85
+# waiting new version
+RESTRICT="test"
+
+#BDEPEND="test? (
+#	dev-python/numpy[${PYTHON_USEDEP}]
+#	dev-python/pdfrw[${PYTHON_USEDEP}]
+#	dev-python/pikepdf[${PYTHON_USEDEP}]
+#	dev-python/pytest[${PYTHON_USEDEP}]
+#	dev-python/scipy[${PYTHON_USEDEP}]
+#	<media-gfx/imagemagick-7.0.0[jpeg,jpeg2k,png,q8,q32,tiff]
+#)"
+RDEPEND="dev-python/pillow[${PYTHON_USEDEP}]"
+
```

Comment:
```
>: img2pdf-gui 
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.8/img2pdf-gui", line 33, in <module>
    sys.exit(load_entry_point('img2pdf==0.4.0', 'gui_scripts', 'img2pdf-gui')())
  File "/usr/lib/python3.8/site-packages/img2pdf.py", line 2371, in gui
    import tkinter
ModuleNotFoundError: No module named 'tkinter'
```
Is this a problem? Do we have tkinter in Gentoo? Not sure should you try to package it or remove the *-gui* executable :\

Reply:
> Do we have tkinter in Gentoo?

Haven't tested but USE=tk on python should provide it
https://wiki.gentoo.org/wiki/Tkinter

Reply:
Yep, that did it :)

I guess you could include a `gui` USE flag and I guess this should work then:
`gui? ( $(python_gen_impl_dep tk) )`
(But if it is too hard just ignore for now)

Reply:
Thanks @ionenwks and @juippis.
Sorry for mess and the little care I took in testing for this package, I am not the proxy maintainer and I bumped only for add it as ocrmypdf dependency in my overlay.

Reply:
Eeh, it's fine. I hope you understand we rather get it as close to perfect in here than deal with bug reports later on.

I'm fine with dropping `~x86`, I seriously have doubts how popular that arch is anymore.